File: demos/bookDemos/Introduction/sigmoidPlot.m
%% Plot the sigmoid function
%
%%

% This file is from pmtk3.googlecode.com

xs = -10:0.1:10;
p = 1./(1+exp(-xs));
figure;
plot(xs, p, '-', 'linewidth', 3)
printPmtkFigure('sigmoidPlot')

File: python/demos/sigmoidPlot.py
#!/usr/bin/env python

import numpy as np
import matplotlib.pylab as pl

e = np.exp(1)
x = np.linspace(-10, 10, 1000)
y = e**x / (e**x + 1)
pl.plot(x, y)
pl.title('sigmoid function')
pl.savefig('sigmoidPlot.png')
pl.show()
